### DISCUSSION TITLE
where is the “Automatic mesh optimization” in src?

### DISCUSSION THREAD

**Discussion by for-heart-peace** [cnr-isti-vclab/meshlab#988] [Q&A]
The "Automatic mesh optimization" in https://www.meshlab.net/ is a so solid work. I need learn it by code, and could you tell me the specific location of the code and the paper that the algorithm refers to.
Thank you!

**Comment by alemuntoni** [comment]
"Automatic mesh optimization" is not a single tool in MeshLab. Lot of filters in meshlab allow to perform automatic mesh optimization.
